ci/pipeline.yml

```diff
@@ -0,0 +1,85 @@
+---
+jobs:
+- name: deploy-production
+  plan:
+  - get: cg-docs-source
+    trigger: true
+  - task: build
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: jojomi/hugo
+          tag: '0.16'
+      inputs:
+      - name: cg-docs-source
+      outputs:
+      - name: cg-docs-compiled
+      run:
+        dir: cg-docs-source
+        path: sh
+        args:
+        - -exc
+        - |
+          # install dependencies
+          # git required for NPM, the rest required for node-sass->node-gyp
+          apk add --no-cache --update g++ git make nodejs python
```

Comment:
@afeld @dlapiduz , tested locally, and found `node-sass` isn't an actual dependency it was probably considered one early in development but then we found we could build the docs site without compiling from sass. If you remove `node-sass` from `devDependencies` then python, make g++ shouldn't be required. Hope that helps a bit.